06-building-a-program-start-to-finish/writing-tests/Output/test/spec/test.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
+const expect = chai.expect;
+ 
+ const InvertedIndex = require('../../src/js/inverted-index.js');
+ const books = require('../books.json');
+ 
+ const emptyBook = [];
```

Comment:
I'll like to see you get an actual empty JSON file rather than declare an empty array.